tests/subscriber.coffee

```diff
@@ -179,19 +179,19 @@ describe 'Subscriber', ->
         it 'should not remove subscription on an unexisting subscription', (done) =>
             subscriber = new Subscriber(@subscriber.redis, 'invalidid')
             subscriber.removeSubscription @testEvent, (removed) =>
-                should.not.exist removed
+                should.exist(removed)
```

Comment:
No, that is correct. Take a look at the code (and the PR message). A successful removal now returns null (so as not to break async.serial). Otherwise it returns "Not exists". So this test was updated accordingly.